wp-content/plugins/core/src/Blocks/Types/Accordion/Accordion.php

```diff
@@ -47,10 +47,12 @@ public function add_block() {
 							'Cras ut ornare dui, sed venenatis est. Donec euismod in leo quis consequat.',
 							'tribe'
 						),
-						self::CTA         => [
-							'title'  => esc_html__( 'Lorem ipsum', 'tribe' ),
-							'url'    => '#',
-							'target' => '',
+						CTA::GROUP_CTA    => [
+							CTA::LINK => [
+								'title'  => esc_html__( 'Lorem ipsum', 'tribe' ),
```

Comment:
Did a quick test. Not sure this content surfaces anywhere. If that's the case, I'd rename "Call to Action Link" so we don't have random non-semantic text floating around.

Reply:
What do you mean by surface anywhere? I can definitely rename the title to that though, some had `Lorem ipsum` and some had `Call to Action Link`. This is for the "example" that appears for when you "browse all blocks" and then hover over the block. Let me know what you prefer and I will update so they are all the same.

Reply:
Ah I see it in the block select preview, but not in the live preview. #TIL

Disregard this message.